lib/frontman/app.rb

```diff
@@ -131,12 +131,18 @@ def add_asset_pipeline(
     end
 
     sig { params(file_path: String).void }
-    def import_config(file_path)
+    def require(file_path)
```

Comment:
Can we keep the method name `import_config` and not add `require`? This is a bc breaking change that I'm not comfortable making, as it will break all current `require 'gemname'` statements in configuration files 🙂

Reply:
Should we maybe rename global_require to require then?

Reply:
I would prefer that indeed. The variant that's scoped to the `Frontman::App` instance can be called `local_require`, `app_require`, or something similar. WDYT?